core/vm/evm.go

```diff
@@ -49,15 +49,11 @@ func run(evm *EVM, contract *Contract, input []byte, readOnly bool) ([]byte, err
 	// Intercept the StateManager calls
 	if contract.Address() == StateManagerAddress {
 		log.Debug("Calling State Manager contract.", "StateManagerAddress", hex.EncodeToString(StateManagerAddress.Bytes()))
-		gas := stateManagerRequiredGas(input)
-		if contract.UseGas(gas) {
-			ret, err := callStateManager(input, evm, contract)
-			if err != nil {
-				log.Error("State manager error!", "Error", err)
-			}
-			return ret, err
+		ret, err := callStateManager(input, evm, contract)
+		if err != nil {
+			log.Error("State manager error!", err)
```

Comment:
Missing the `Error` key here before the `err`. It looks like in this case it'll append `nil` and the formatting will look weird.

See: https://github.com/ethereum-optimism/go-ethereum/blob/fa42b782283e5ed9d1af811f53d1d92b22bd48e5/log/logger.go#L209